conftest.py

```diff
@@ -28,7 +21,20 @@
 os.mkdir(os.path.join(os.environ['XDG_CONFIG_HOME'], 'astropy'))
 os.mkdir(os.path.join(os.environ['XDG_CACHE_HOME'], 'astropy'))
 
+try:
+    from pyvo import __version__ as version
+except ImportError:
+    version = 'unknown'
+
+TESTED_VERSIONS['pyvo'] = version
+
+
 # Note that we don't need to change the environment variables back or remove
 # them after testing, because they are only changed for the duration of the
 # Python process, and this configuration only matters if running pytest
 # directly, not from e.g. an IPython session.
+
+from astropy.utils.iers import conf as iers_conf
+
```

Comment:
iers is a huge pile of pain.
This is a workaround to make it avoid trying to download the table when running the tests locally (it would error out with remote data stuff).

So, yes, I can dig up some explanation about this, I'm pretty sure I inherited the workaround from other places.

Reply:
Would it be enough to move the comment from the line below to before the import?

Reply:
I don't think we need a long explanation. Something that will help us remember what is all about and possibly trace it, so we, or other maintainers after us can tell when it's safe to remove or change it.

Reply:
This is my last comment. I could merge it afterwards unless you want Tom's feedback as well.